packages/engine-api-client/src/metrics/index.ts

```diff
@@ -0,0 +1,23 @@
+import {IGauge, IHistogram} from "@lodestar/utils/metrics";
```

Comment:
Move packages/engine-api-client/src/metrics/index.ts to packages/engine-api-client/src/metrics.ts